src/driver/landing_pages.mli

```diff
@@ -1,5 +1,18 @@
+type u = unit
```

Comment:
What is the point of this ?

Reply:
Odoc_unit exposes a type 'unit' that's not `()`! Perhaps I should just fix that in this PR, and avoid this mess.